CAM/SetCamFarDof.md

```diff
@@ -8,8 +8,9 @@ ns: CAM
 void SET_CAM_FAR_DOF(Cam cam, float farDOF);
 ```
 
+Specifies when the camera should stop being in focus.
 
 ## Parameters
 * **cam**: 
-* **farDOF**: 
+* **farDOF**: Distance in in standard units
```

Comment:
I've linked some of the other natives related and my example on the cfx forums

Reply:
Understood, it's better to have the examples here though, what if the forum URL ever changes or the thread gets removed? It would end up being a  broken link.

Reply:
good point, will copy it from the thread